tex/set-theory/forcing.tex

```diff
@@ -355,21 +393,29 @@ \section{Names, and the generic extension}
 			\varnothing & \text{otherwise}.
 		\end{cases}
 	\]
-	In particular, remembering that $G$ is nonempty we see that
+	In particular, since $1_\Po \in G$, then:
+	\begin{itemize}
+		\ii when $n = 0$, $\tau = \varnothing$, so $\tau^G = \varnothing$.
+		\ii when $n = 1$ and $p_1 = 1_\Po$, $\tau = \{ \langle \varnothing, 1_\Po \rangle \}$,
+		so $\tau^G = \{ \varnothing \}$.
+	\end{itemize}
+	So,
 	\[ \left\{ \tau^G \mid \tau \in \Name_2 \right\} = V_2. \]
 	In fact, this holds for any natural number $n$, not just $2$.
 \end{example}
 So, $M[G]$ and $M$ agree on finite sets.
 
 Now, we want to make sure $M[G]$ contains the elements of $M$.
-To do this, we take advantage of the fact that $1_\Po$ must be in $G$, and define
+The proof of $\left\{ \tau^G \mid \tau \in \Name_2 \right\} = V_2$ above can be easily adapted:
+Since $1_\Po$ must be in $G$, we define
 for every $x \in M$ the set
 \[ \check x = \left\{ \left<\check y, 1_\Po\right> \mid y \in x \right\} \]
 by transfinite recursion.
-Basically, $\check x$ is just a copy of $x$ where we add check marks and tag every element with $1_\Po$.
+Basically, $\check x$ is just a copy of $x$ where we tag every element \emph{at every nesting level}
+with $1_\Po$.
 
 \begin{example}
-	Compute $\check 0 = 0$ and $\check 1 = \left\{ \left<\check 0, 1_\Po\right> \right\}$.
+	Compute $\check 0 = 0$ and $\check 1 = \{ \langle \check 0, 1_\Po \rangle \}$.
```

Comment:
in my opinion, originally the `⟨…⟩` seems to be unnecessarily enlarged, just because of the check above the items.

Maybe https://tex.stackexchange.com/questions/173717/is-it-ever-bad-to-use-left-and-right .